compiler/executable/match_/planner/plan.rs

```diff
@@ -214,6 +227,7 @@ impl VertexId {
 #[derive(Clone)]
 pub(super) struct ConjunctionPlanBuilder<'a> {
     shared_variables: Vec<Variable>,
+    required_inputs: Vec<Variable>,
```

Comment:
I'm going to approve and merge, but can you update these names too please, perhaps in your next PR?